clients/web/test/spec/sortWidgetSpec.js

```diff
@@ -33,7 +33,7 @@ describe('Sort user list', function () {
         girderTest.goToUsersPage()();
         runs(function () {
             expect($('.g-user-list-entry').length).toBe(3);
-            expect($('a.g-user-link').text()).toBe('AFirstName ALastNameBFirstName BLastNameCFirstName CLastName');
+            expect($('a.g-user-link').text()).toMatch(/.*AFirstName ALastNameBFirstName BLastNameCFirstName CLastName.*/);
```

Comment:
Probably most robust to add a `.*` in between them in case we add more data that happens to fall between the values.